app/controllers/rentals_controller.rb

```diff
@@ -23,20 +34,42 @@ def index
   # GET /rentals/1
   def show; end
 
-  # GET /rentals/cost?end_time=time&start_time=time&item_type=1
+  # GET /rentals/cost?end_time=time&start_time=time&item_types=...
   def cost
-    start_time = Time.zone.parse(params[:start_time]).to_date.to_s
-    end_time = Time.zone.parse(params[:end_time]).to_date.to_s
-    item_type = ItemType.find(params[:item_type]) if params[:item_type]
-    render json: Rental.cost(start_time, end_time, item_type)
+    _params = cost_params
```

Comment:
You should only preface a variable with an underscore when you're not using it. This is used for when you're pattern-matching something into multiple variables, but only need one. For instance:

```ruby
a, _b = some_method_that_returns_a_tuple

hash.each_pair do |key, _value|
  i_only_want_the(key)
end
```